pkgs/development/tools/kustomize/default.nix

```diff
@@ -0,0 +1,28 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
+{ lib, stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "kustomize-${version}";
+  version = "1.0.4";
+
+  goPackagePath = "github.com/kubernetes-sigs/kustomize";
+
+  src = fetchFromGitHub {
+    sha256 = "0lbf94wz34axaf8ps7h79qbj4dpihrpvnqa12zrawcmmgqallwhm";
+    rev = "v${version}";
+    repo = "kustomize";
+    owner = "kubernetes-sigs";
+  };
+
+  meta = with lib; {
+    description = "Customization of kubernetes YAML configurations";
+    longDescription = ''
+      kustomize lets you customize raw, template-free YAML files for
+      multiple purposes, leaving the original YAML untouched and usable
+      as is.
+    '';
+    homepage = "https://github.com/kubernetes-sigs/kustomize";
```

Comment:
URLs don't need quotes around them in Nix, can you remove them?

Reply:
Didn't know that. Will do so 😄